extensions/amp-amazon-onetag/validator-amp-amazon-onetag.protoascii

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-amazon-onetag
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-amazon-onetag"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-amazon-onetag>
+  html_format: AMP
+  tag_name: "AMP-AMAZON-ONETAG"
```

Comment:
Your test examples show a few more attributes: `marketplace` and `instanceid`.  I'm guessing these are intended to be valid. Can you add them in the config for this tag. For example:

```
attrs: {
  name: "instanceid"
}
attrs: {
  name: "marketplace"
}
```